app/models/pageflow/feed_entries_query.rb

```diff
@@ -0,0 +1,19 @@
+module Pageflow
+  class FeedEntriesQuery < ApplicationQuery
```

Comment:
On second thought, a way less technical name for this might just be `EntryFeed`. `resolve` can then be `entries`. Let's return `PublishedEntries` there directly instead of having the decorator creation inside the view code.